src/midi/midi_mt32.cpp

```diff
@@ -683,101 +694,124 @@ void MidiHandler_mt32::Close()
 
 	// Reset the members
 	service.reset();
-	total_buffers_played = 0;
-	last_played_frame = 0;
 
-	is_open = false;
+	last_rendered_ms = 0.0;
+	ms_per_frame     = 0.0;
```

Comment:
As per above; "frame" is fine when we're in the context of a pure sample-based audio class, e.g. the mixer, but in the context of MIDI "frame" has multiple meanings.